src/infrastructure/todoapp-pod.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Pod
+metadata:
+ name: todoapp:3.0.0
```

Comment:
The `name` field should not contain a version number. It should simply be `todoapp`. The version should be part of the image tag, not the Pod name.